variables/_svgs.pcss

```diff
@@ -257,3 +332,28 @@
 		fill: var(--color-icon-active);
 	}
 }
+
+/* -----------------------------------------------------------------------------
+ * SVG - Search
+ * ----------------------------------------------------------------------------- */
+
+@svg-load search url('../icons/search.svg') {
```

Comment:
we already have search defined above.

Reply:
🤦‍♂ my bad, been pushing changes to many places and it seems this was a copy and paste that I never wiped.